com.unity.render-pipelines.universal/Runtime/Deprecated.cs

```diff
@@ -1,11 +1,48 @@
-﻿// This file should be used as a container for things on its
+// This file should be used as a container for things on its
 // way to being deprecated and removed in future releases
 
+using System;
+using UnityEngine.Scripting.APIUpdating;
+
 namespace UnityEngine.Rendering.Universal
 {
     public abstract partial class ScriptableRenderPass
     {
         // This callback method will be removed. Please use OnCameraCleanup() instead.
         public virtual void FrameCleanup(CommandBuffer cmd) => OnCameraCleanup(cmd);
     }
+
+    /// <summary>
+    /// Default renderer for Universal RP.
+    /// This renderer is supported on all Universal RP supported platforms.
+    /// It uses a classic forward rendering strategy with per-object light culling.
+    /// </summary>
+    [Obsolete("ForwardRenderer has been deprecated. Use StandardRenderer instead (UnityUpgradable) -> StandardRenderer", true)]
+    public sealed class ForwardRenderer: ScriptableRenderer
+    {
+        public ForwardRenderer(ForwardRendererData data) : base(data)
+        {
+            throw new NotSupportedException("ForwardRenderer has been deprecated. Use StandardRenderer instead");
```

Comment:
why not introduce a constant with this message and reuse it everywhere?

example: 
https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxASwDZICYgNQA+AAgEwCMAsAFDUDaA8sFAPYZzwAUAsnFFBAHM4ASgC61IgGYABKWkBhadQDe1aetkyAxswB2sWWQAM0nn0FxpAXmkAiABJwMGZrYDcajZ/UMmrdnDcvPxCYt6asgAsphzCSlQa0qoJiRowABYIzADu0rpwuQByzDAAygCuAA6VzAjw2ACiAB5acJUwaHpB5qEeKeoAvtQDQA=

Reply:
Refactored to constant.